front/src/views/user/entertainments-new/entertainments-new.view.css

```diff
@@ -0,0 +1,98 @@
+.new-entertainments, .autocomplete-container, .manage-container {
+  display: flex;
+}
+
+.title {
+  font-weight: normal;
+  margin-bottom: 30px;
+}
+
+.new-entertainments {
+  max-width: 1095px;
+  height: 90vh;
+  justify-content: space-between;
+}
+
+.gmap-container {
+  width: 100%;
+  height: 100%;
+  margin: 0 20px 0 40px;
+}
+
+.new-entertainments-map {
+  width: 100%;
+  height: 100%;
+}
+
+.new-entertainments-autocomplete {
+  width: 94%;
+  height: 50px;
+  background-color: #fff;
+  padding:0 20px;
+}
+
+.autocomplete-container >>> .set-marker {
+  border-radius: 0;
+  min-width: 60px;
+  padding:0;
+  box-shadow:none;
+  height: 50px;
+}
+
+.note-field {
+  margin-top: 40px;
+}
+
+.person-count-field {
+  margin-bottom: 10px;
+}
+
+.mmt-btn.v-btn:not(.v-btn--flat):not(.v-btn--text):not(.v-btn--outlined) {
+  background-color: var(--continue-button-background-color);
+}
+
+.mmt-btn.v-btn:not(.v-btn--disabled):not(.v-btn--flat):not(.v-btn--text):not(.v-btn--outlined) >>> .v-btn__content {
+  color: var(--continue-button-active-color);
+}
+
+.mmt-btn {
+  padding:10px 48px;
+  margin: 14px 0;
+  color: white;
+}
```

Comment:
might, create one more type of button and styles for it in common styles?
like text-button or other one?